src/components/Authentication/Registration/FormField.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input } from 'semantic-ui-react';
+
+const FormField = ({
+  onChange, inputName, error, type,
+}) => {
+  const convertToUpperCase = string => string.replace(/^[a-z]/, x => x.toUpperCase());
+  const dataTestAttr = string => `${string}-error`;
+
+  return (
+    <Form.Field>
+      <label htmlFor={inputName}>{convertToUpperCase(inputName)}</label>
+      <Input
+        onChange={onChange}
+        type={type}
+        name={inputName}
+        placeholder={convertToUpperCase(inputName)}
+      />
+      {error[inputName] ? error[inputName].map(err => (
+        <small key={inputName} data-test={dataTestAttr(inputName)} id={inputName} className="error">{err}</small>
```

Comment:
Use pascal case naming for `data-test`.

Reply:
I was using a custom attribute for testing purposes to get that component.

Reply:
Understood.